src/main/groovy/life/qbic/datamodel/datasets/datastructure/files/maxquant/AllPeptides.groovy

```diff
@@ -0,0 +1,30 @@
+package life.qbic.datamodel.datasets.datastructure.files.maxquant
+
+import life.qbic.datamodel.datasets.datastructure.files.DataFile
+
+/**
+ * A specialisation of a DataFile, represents an MaxQuant AllPeptides file
+ *
+ * @since 2.10.0
+ *
+*/
+class AllPeptides extends DataFile{
+    final private static String FILE_TYPE = "txt"
+
+    final private static String NAME_SCHEMA = $/allPeptides.*/$
+
+    protected AllPeptides(String name, String relativePath) {
+        super(name, relativePath, FILE_TYPE)
+        validateName()
+    }
+
```

Comment:
Since this is a public method it would make sense to include javadocs akin to the nfcore datafiles